src/host/toolsHost.ts

```diff
@@ -92,10 +92,6 @@ export class ToolsHost {
         encodeMessageForChannel(msg => vscode.postMessage(msg, '*'), 'getVscodeSettings', {id});
     }
 
-    sendToVscodeOutput(consoleMessage: string): void {
```

Comment:
Should we remove the polyfills that reference this too then? E.g. in `src/host/polyfills/simpleView.ts` there's a call to `Host.InspectorFrontendHost.sendToVscodeOutput(message);` (also has a related test)

Reply:
I thought about it, but since the polyfill system should be removed soon, I was trying to avoid touching it

Reply:
Or perhaps more broadly, is this the time to just purge `src/host` entirely? Or would you prefer to save that for a separate follow-up?

Reply:
I'd like that to be a separate follow up, so that removing this feature is separate from removing the polyfill system in case a revert is needed